apps/chatbot/src/app/main.py

```diff
@@ -6,35 +6,41 @@
 import uuid
 import boto3
 import datetime
+from boto3.dynamodb.conditions import Key
 from botocore.exceptions import BotoCoreError, ClientError
 from fastapi import FastAPI, HTTPException
 from starlette.middleware.cors import CORSMiddleware
 from pydantic import BaseModel
 
 from src.modules.chatbot import Chatbot
 
-
 params = yaml.safe_load(open("config/params.yaml", "r"))
 prompts = yaml.safe_load(open("config/prompts.yaml", "r"))
 
+AWS_DEFAULT_REGION = os.getenv('CHB_AWS_DEFAULT_REGION', os.getenv('AWS_DEFAULT_REGION', None))
+
 class Query(BaseModel):
   question: str
   queriedAt: str | None = None
 
 if (os.getenv('environment', 'dev') == 'local'):
   profile_name='dummy'
   endpoint_url='http://localhost:8000'
+  region_name = AWS_DEFAULT_REGION
 
 boto3_session = boto3.session.Session(
-  profile_name = locals().get('profile_name', None)
+  profile_name = locals().get('profile_name', None),
+  region_name=locals().get('region_name', None)
```

Comment:
In the lambda, the environment variable `CHB_AWS_DEFAULT_REGION` contains `eu-west-3`. For this reason, the lambda would look for the dynamodb table in that region instead of the actual default region `eu-south-1`

```suggestion
```

Reply:
profile_name, region_name and endpoint_url exists only in `local` environment